app/routes/flush.py

```diff
@@ -76,7 +78,15 @@ def _submit_data(user):
         full_routing_path = router.full_routing_path()
 
         message = json_dumps(
-            convert_answers(
+            convert_answers_v2(
+                schema,
+                questionnaire_store,
+                full_routing_path,
+                submitted_at,
+                flushed=True,
+            )
+            if metadata.version is AuthPayloadVersion.V2
+            else convert_answers(
```

Comment:
Minor, instead of defining the function argument twice, work out the method to call upfront. i.e

```py
        converter = (
            convert_answers_v2
            if metadata.version is AuthPayloadVersion.V2
            else convert_answers
        )

        message = json_dumps(
            converter(
                schema,
                questionnaire_store,
                full_routing_path,
                submitted_at,
                flushed=True,
            )
        )
```

Reply:
Hi Mebin - I agree but I was not able create another local as the lint fails as the  methods seems to have too may locals. Allowed is fifteen, me adding this will make it 16 and will have to apply an exception. I also like this style :-) makes it easier to understand whats happening :-)

Reply:
Hey Amit, I think we address it otherwise, we are leaving the problem to the next person to deal with. Parts of it could be refactored into method or in this case I am happy to ignore it. Will leave it to you on whether you refactor or ignore the warning.

Reply:
👍  refactored!